src/test/kotlin/org/rust/ide/refactoring/RsInlineFunctionTest.kt

```diff
@@ -1071,6 +1071,72 @@ class RsInlineFunctionTest : RsTestBase() {
 
             """)
 
+    fun `test inline function from different module with two single use statements`() = doTest("""
+            use foo::bar::test;
+            use foo::bar::test2;
+            fn main() {
+                test();
+                test2();
+            }
+            mod foo {
+                pub mod bar {
+                    pub fn /*caret*/test() {
+                        println!("test");
+                    }
+                    pub fn test2() {
+                        println!("test2");
+                    }
+                }
+            }""", """
```

Comment:
Let's move quotes on the new line (just like in other tests)
```suggestion
        }
    """, """
```